src/core/frameworks/gql/queries.ts

```diff
@@ -314,45 +310,104 @@ export const GET_BORROWER_POSITIONS_QUERY = gql`
   ${SPIGOT_EVENT_FRAGMENT}
   ${ESCROW_FRAGMENT}
 
-  query getBorrowerPositions($borrower: String!) {
-    lineOfCredits(where: { borrower_contains: $borrower }) {
-      ...BaseLineFrag
+  fragment PositionsFrag on LineOfCredit {
```

Comment:
isnt this the same as LinePage query?

Reply:
I see what you mean. 

I've simplified the `GET_LINE_PAGE_QUERY` by removing the repetitive code in the query and replacing it with the newly created `LineOfCreditsFrag`.